pkg/v1/providers/ytt/lib/helpers.star

```diff
@@ -142,6 +142,12 @@ def validate_oidc():
   end
 end
 
+def validate_flags_combination():
+  if data.values.TKG_CLUSTER_ROLE == "management" and data.values.IS_WINDOWS_WORKLOAD_CLUSTER:
+    assert.fail("The IS_WINDOWS_WORKLOAD_CLUSTER option can't be used with a cluster role different of workload.")
```

Comment:
nits: 
the phrasing of the error message is a little wonky. How about
"The IS_WINDOWS_WORKLOAD_CLUSTER option can only be used when creating a workload cluster"
The function name is quite general. Did you mean this for function to be used in the future for validating other combinations of config values? If so, can you add a comment block to describe its purpose. Maybe something like validate_configuration_combinations() would be more appropriate, and consistent with the validate_configuration